emission/integrationTests/nominatimTests/nominatim-docker-test.yml

```diff
@@ -0,0 +1,34 @@
+name: nominatim-docker-test
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches:
+        - master
+        - nominatim
```

Comment:
we may be OK with running it on every push to master, but let's take it off for now to reduce our GitHub actions usage. Also, there is no `nominatim` branch in the main repo so even if we restored on push, we would not include the `nominatim` branch here